### PR TITLE
Updated ImageView to implement AutoPropertyDiffable

### PR DESCRIPTION
**Pull request by aducret**
## Screencast
pr

### CODE REVIEW

**Comment by aducret** [comment]
[Hacer que ImageView sea diffable en Portal](https://trello.com/c/uU9E3PC4/301-hacer-que-imageview-sea-diffable-en-portal)

File: Example/Views/ModalScreen.swift [hunk 1]
```diff
@@ -44,6 +44,17 @@ enum ModalScreen {
                             textView.textFont = Font(name: "Helvetica")
                             textView.textSize = 18
                         }),
+                    imageView(
+                        image: Image(image: UIImage(named: "test")!),
```
Comment (guidomb) [inline Example/Views/ModalScreen.swift]: Use [this](https://github.com/guidomb/Portal/blob/4d411844c57d6652e9960bc2b3dd92fc50a3f2ec/Portal/View/UIKit/UIKitRenderable.swift#L21) method to load images

File: Example/Views/ModalScreen.swift [hunk 2]
```diff
@@ -44,6 +44,17 @@ enum ModalScreen {
...
+                        style: styleSheet {
+                            $0.backgroundColor = .yellow
+                        },
+                        layout: layout {
+                            $0.height = Dimension(value: 50)
+                            $0.width = Dimension(value: 50)
+                            $0.alignment = Alignment(self: .center)
```
Comment (guidomb) [inline Example/Views/ModalScreen.swift]: use

```swift
aligment() {
  $0.`self` = .center
}
```

**Review by guidomb — CHANGES_REQUESTED** [review]

File: Example/Views/ModalScreen.swift (hunk 1 above)
Comment (aducret) [reply Example/Views/ModalScreen.swift]: What is the difference? if we expect the users only use `loadImage` method, the image's init should be private?

File: Example/Views/ModalScreen.swift (hunk 2 above)
Comment (aducret) [reply Example/Views/ModalScreen.swift]: same question as in the previous comment.

File: Example/Views/ModalScreen.swift (hunk 1 above)
Comment (guidomb) [reply Example/Views/ModalScreen.swift]: The main reason behind using `Image.loadImage` is not to couple user code with UIKit. Image init should be `internal`. In a future refactor we should change this, we should load image in a declarative view, loading the image should be done by the renderer.

File: Example/Views/ModalScreen.swift (hunk 2 above)
Comment (guidomb) [reply Example/Views/ModalScreen.swift]: Yes, all layout objects that have a builder function should have `internal` inits.

**Review by guidomb — APPROVED** [review]

**Pull request merged**